src/Processor/ProcessorFactory.php

```diff
@@ -51,7 +51,9 @@ class ProcessorFactory
         EventCodes::VOID_PENDING_REFUND => VoidPendingRefundProcessor::class,
         EventCodes::CHARGEBACK => ChargebackProcessor::class,
         EventCodes::CHARGEBACK_REVERSED => ChargebackReversedProcessor::class,
```

Comment:
I couldn't see the updated processor for `CHARGEBACK_REVERSED`. Don't we need to modify it?

Reply:
No we don't have to modify the chargeback_reversed processor as this was already present